### PR TITLE
PXE UI change

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: cfme/tests/services/test_pxe_service_catalogs.py}}

### CODE REVIEW

File: cfme/infrastructure/pxe.py [hunk 1]
```diff
@@ -35,12 +35,12 @@
     fields=[
         ('name_text', Input('name')),
         ('log_protocol', Select("//select[@id='log_protocol']")),
-        ('uri_text', Input('uri')),
```
Comment (psav) [inline cfme/infrastructure/pxe.py]: This just changes order, is that right?

**Comment by dajoRH** [comment]
Lint report for commit 80d587094c0604fda90dbe4347f7a6efd4baafab:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/infrastructure/pxe.py (hunk 1 above)
Comment (sshveta) [inline cfme/infrastructure/pxe.py]: Yes , just changes order hence the URI field gets sufficient time to load else it shows error element not found

**Comment by psav** [comment]
We need a PRT run on this too.

**Comment by dajoRH** [comment]
Lint report for commit 6811d66089b8998c0232d0dd55cfaffee3f5adf8:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/infrastructure/pxe.py (hunk 1 above)
Comment (psav) [inline cfme/infrastructure/pxe.py]: We need to try to find the root cause of this, as unfortunately things may sometimes be filled in quicker than we hope. We are currently trying to work with dev on this.

**Pull request merged**